awx/ui_next/src/screens/InstanceGroup/ContainerGroupDetails/ContainerGroupDetails.jsx

```diff
@@ -1,14 +1,137 @@
-import React from 'react';
-import { Card, PageSection } from '@patternfly/react-core';
+import React, { useCallback } from 'react';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Link, useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { Button, Label } from '@patternfly/react-core';
+
+import { VariablesDetail } from '../../../components/CodeMirrorInput';
+import AlertModal from '../../../components/AlertModal';
+import { CardBody, CardActionsRow } from '../../../components/Card';
+import DeleteButton from '../../../components/DeleteButton';
+import {
+  Detail,
+  DetailList,
+  UserDateDetail,
+  DetailBadge,
+} from '../../../components/DetailList';
+import useRequest, { useDismissableError } from '../../../util/useRequest';
+import { jsonToYaml, isJsonString } from '../../../util/yaml';
+import { InstanceGroupsAPI } from '../../../api';
+
+const Unavailable = styled.span`
+  color: var(--pf-global--danger-color--200);
+`;
+
+function ContainerGroupDetails({ instanceGroup, i18n }) {
+  const { id, name } = instanceGroup;
+
+  const history = useHistory();
+
+  const {
+    request: deleteInstanceGroup,
+    isLoading,
+    error: deleteError,
+  } = useRequest(
+    useCallback(async () => {
+      await InstanceGroupsAPI.destroy(id);
+      history.push(`/instance_groups`);
+    }, [id, history])
+  );
+
+  const { error, dismissError } = useDismissableError(deleteError);
 
-function ContainerGroupDetails() {
   return (
-    <PageSection>
-      <Card>
-        <div>Container group details</div>
-      </Card>
-    </PageSection>
+    <CardBody>
+      <DetailList>
+        <Detail
+          label={i18n._(t`Name`)}
+          value={instanceGroup.name}
+          dataCy="container-group-detail-name"
+        />
+        <Detail
+          label={i18n._(t`Type`)}
+          value={i18n._(t`Container group`)}
+          dataCy="container-group-type"
+        />
+        <Detail
+          label={i18n._(t`Credential`)}
+          value={
+            <Label variant="outline" color="blue">
+              {instanceGroup.summary_fields.credential.name}
+            </Label>
+          }
+          dataCy="container-group-credential"
+        />
+        {instanceGroup.capacity ? (
+          <DetailBadge
+            label={i18n._(t`Used capacity`)}
+            content={`${Math.round(
+              100 - instanceGroup.percent_capacity_remaining
+            )} %`}
+            dataCy="container-group-used-capacity"
+          />
+        ) : (
+          <Detail
+            label={i18n._(t`Used capacity`)}
+            value={<Unavailable>{i18n._(t`Unavailable`)}</Unavailable>}
```

Comment:
Container groups are ephemeral and don't really have dedicated instances in the typical sense. My understanding is that container groups don't have capacity values like instance groups do.

In the 3.8 interface we don't display values for these on the details page so I wanted to confirm. We also don't show any capacity values on the list item, which seems intentional:

![Screenshot from 2020-09-04 13-00-50](https://user-images.githubusercontent.com/9753817/92267485-bb297d00-eeae-11ea-840f-f3806e134621.png)

Reply:
I will verify this one. Thanks.

Reply:
Updated. Please, see pictures above.